airflow-core/src/airflow/api_fastapi/execution_api/datamodels/hitl.py

```diff
@@ -17,13 +17,14 @@
 from __future__ import annotations
 
 from datetime import datetime
-from typing import Any
+from typing import Any, cast
```

Comment:
Why do we need to import "cast"?

Reply:
No problem - still Python's interpreter struggles, please check it out :)